handler.js

```diff
@@ -35,20 +40,32 @@ exports.graphqlHandler = function graphqlHandler(event, context, callback) {
     callback(error, output);
   }
 
-  const handler = server.graphqlLambda({ schema: graphqlSchema });
+  const handler = server.graphqlLambda((event, context) => {
+    const { headers } = event;
+    const { functionName } = context;
 
-  connectToDatabase()
-    .then(() => {
-      return handler(event, context, callbackFilter);
-    })
-    .catch(err => {
-      console.log('MongoDB connection error: ', err);
-      // TODO: return 500?
-      process.exit();
-    });
-};
+    return {
+      schema: schema,
+      context: {
+        headers,
+        functionName,
+        event,
+        context
+      }
+    };
+  });
+
+  try {
+    await connectToDatabase();
+  } catch (err) {
+    log('MongoDB connection error: ', err);
+    // TODO: return 500?
+    process.exit();
+  }
+  return handler(event, context, callbackFilter);
+}
 
-exports.apiHandler = lambdaPlayground({
+export const apiHandler = lambdaPlayground({
```

Comment:
Like above, works with `exports.apiHandler` but this give me a `handler for \`api\` is not a function.